$NetBSD$

* no matter libpinyin tools found or not, we can build it

--- CMakeLists.txt.orig	2013-07-15 20:34:05.000000000 +0000
+++ CMakeLists.txt
@@ -26,6 +26,12 @@ if(NOT DEFINED LIB_INSTALL_DIR)
   set(LIB_INSTALL_DIR ${CMAKE_INSTALL_PREFIX}/lib)
 endif()
 
+if (LIBPINYIN_GEN_UNIGRAM AND LIBPINYIN_GEN_BINARY_FILES AND LIBPINYIN_IMPORT_INTERPOLATION)
+    set(LIBPINYIN_TOOLS_FOUND 1)
+else()
+    set(LIBPINYIN_TOOLS_FOUND 0)
+endif ()
+
 set(FCITX_LIBPINYIN_ZHUYIN_DATADIR
   "${FCITX4_PREFIX}/share/fcitx/libpinyin/zhuyin_data")
 set(libdir ${LIB_INSTALL_DIR})
